dashboard/src/components/AppView/ResourceTable/ResourceItem/SecretItem/SecretItemDatum.v2.tsx

```diff
@@ -13,11 +16,29 @@ interface ISecretItemDatumProps {
 
 function SecretItemDatum({ name, value }: ISecretItemDatumProps) {
   const [hidden, setHidden] = React.useState(true);
+  const [copied, setCopied] = React.useState(false);
   const toggleDisplay = () => setHidden(!hidden);
+  const setCopiedTrue = () => {
+    setCopied(true);
+    setTimeout(() => {
+      setCopied(false);
+    }, 1000);
+  };
   const decodedValue = atob(value);
 
   return (
     <Row>
+      <Column span={5}>
+        <div className="secret-datum-text">{name}</div>
+      </Column>
+      <Column span={5}>
+        <input
```

Comment:
Note that I couldn't use the `Input` component, the style was not properly applied :/ 

![Screenshot from 2020-07-23 13-05-37](https://user-images.githubusercontent.com/4025665/88280664-7ffc3180-cce6-11ea-81bf-ce95171c8a97.png)

Reply:
Np. I think this is a special case of Input usage